com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/context.py

```diff
@@ -31,7 +31,7 @@ def delete_json(fn):
 # SPL, the toolkit, the bundle and submits it to the relevant
 # environment
 #
-def submit(ctxtype, graph, config = None, username = None, password = None):
+def submit(ctxtype, graph, config = None, username = None, password = None, resource_url = None):
```

Comment:
Just wondering why we have specific arguments for username, password and resource_url rather than passing them as config items, since config is the configuration for the submission.

`resource_url` may be the wrong term, given that 'resource' has a specific meaning in IBM Streams (a host).

`streamtool man geturl` has this for `--api`

Specifies to print the URL for the REST API web server.